src/navigation/Link/Link.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import styles from "./Link.module.css";
+import { Plus } from "../../icons";
+import classNames from "classnames/bind";
+const cx = classNames.bind(styles);
+
+interface LinkProps {
+  to: string;
+  id?: string;
+  target?: "_blank" | "_self" | "_parent" | "_top";
+  disabled?: boolean;
+  onClick?: (event: React.MouseEvent<HTMLAnchorElement, MouseEvent>) => void;
+  children: React.ReactNode;
+}
+
+const Link: React.FC<LinkProps> = ({
+  to = "www.google.pl",
+  id = "google",
+  target = "_blank",
+  disabled = false,
```

Comment:
**issue:** I don't see that disabled is used anywhere to prevent opening link / click